### DISCUSSION TITLE
NVME Plotting once effect.

### DISCUSSION THREAD

**Discussion by scotchegg** [Chia-Network/chia-blockchain#1475] [Q&A]
I understand all SSDs are not built the same and that you don't want to use your OS as the plotting drive. However I also understand the plot is made only once, after which you no longer use the NVME drive and fall back to in my case the 14TB usb3 drive for answering requests.

Does this one time plotting use have such a detrimental effect on the drive? 

Thanks for clarity before I order another drive...

**Comment by hoffmang9** [comment]
Each k32 uses either 1.6TiBW or 1.8TiBW so you can do the math on how many various NVMe drives can handle. The SSD Endurance document has lots of details - https://github.com/Chia-Network/chia-blockchain/wiki/SSD-Endurance

**Reply by scotchegg** [reply]
Thanks Gene. Sorry to be clear , so once plotted the required spaces for a 12 TB drive the NVMe drive is no longer used or under stress?

Can I also clarify, do we set the temp drive/ space on the fast NVMe drive or theslower 12tb drive?

Thanks

**Reply by hoffmang9** [reply]
Once you've created an e.g. 101.4GiB plot you don't need NVMe SSD at all to farm it - aka keep it in the competition for future farming rewards.

I personally recommend completing all steps of the plotting process on the fast NVMe and then copying the final .plot file to your real storage destination with the OS or command line tools.
